src/actions/movie.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
```

Comment:
I would merge `src/actions/movie.js` + `src/actions/movies.js` it doesn't not need to be one action per screen. As you just have few for now, having just one file would make things easier also merging with filters.